backend/src/rt/token.js

```diff
@@ -0,0 +1,68 @@
+import config from '../config'
+import rp from 'request-promise'
+
+export async function rtLogin(username, password) {
+  console.log(username, password)
+  const { host, authPath } = config.services.rt
+  const res = await rp({
+    method: 'post',
+    uri: `${host}${authPath}`,
+    headers: {
+      accept: 'application/vnd.api+json',
+      'content-type': 'application/vnd.api+json'
+    },
+    body: {
+      data: {
+        type: 'obtain-json-web-tokens',
+        id: null,
+        attributes: {
+          username,
+          password
+        }
+      }
+    },
+    json: true
+  })
+
+  return res.data.token
+}
```

Comment:
Couldn't you just use plain JSON instead of JSON API?

Reply:
the backend enforces `vnd.api+json`. is it worth to enable it for the login? i dont think jsonapi its too bad.

Reply:
For the login JSON API does not make sense. You should be able to just enable it.